fortune.py

```diff
@@ -628,8 +636,8 @@ def format_epilog(self, epilog):
                 min_length=minlength,
                 max_length=maxlength) )
 
-    except ValueError, msg:
-        print >> sys.stderr, msg
+    except (ValueError):
+        print (sys.stderr)
```

Comment:
that looks wrong

Reply:
Not sure how this works. I'm not a python expert

Reply:
I think you mean `print(msg, file=sys.stderr)`. Printing the stderr file handle doesn't really make sense ;-)

Reply:
Thx!